src/atoms/Text/Text.web.js

```diff
@@ -1,3 +1,139 @@
-const Text = () => null;
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import baseTheme from '../../tokens/theme';
+import isDefined from '../../_helpers/isDefined';
+import { getColorKeys, getColor, getLineHeight } from '../../_helpers/theme';
+
+/*
+ * Sets height for the div
+ * @param {Object} theme
+ * @param {String} size
+ * @param {number} maxLines
+ */
+const calculateHeight = ({ theme, size, maxLines }) => {
+  const lineHeight = getLineHeight(theme, size);
+  const containerHeight = `${parseFloat(lineHeight) * maxLines}px`;
+  return containerHeight;
+};
+
+const styles = {
+  fontFamily({ theme, _weight }) {
+    return theme.fonts.family.lato[_weight];
+  },
+  fontSize({ theme, size }) {
+    return theme.fonts.size[size];
+  },
+  color({ theme, color }) {
+    return getColor(theme, color);
+  },
+  align({ align }) {
+    switch (align) {
+      case 'left':
```

Comment:
keep it sepate. Short circuits operations are hard to read

```js
switch (align) {
case left:
//styles
case right:
//styles
case center:
//styles
case default:
//styles
}
```

Reply:
sorry i didnt get this, separate as in ?

Reply:
don't do this
```js
switch (align) {
case left:
case default:
//styles
```
Basically don't club 2 cases

Reply:
done